test/Microsoft.AspNetCore.Mvc.Core.Test/Routing/GlobalRoutingUrlHelperTest.cs

```diff
@@ -93,6 +93,24 @@ public void RouteUrl_WithRouteName_UsesSuppliedValue_OverridingAmbientValue()
             Assert.Equal("/api/orders/10", url);
         }
 
+        [Fact]
+        public void RouteUrl_DoesNotGenerateLink_ToEndpointsWithSuppressLinkGeneration()
```

Comment:
Are there any Razor pages tests we need enable for this?

Reply:
Yes, there are some Razor page functional tests https://github.com/aspnet/Routing/issues/656#issue-344147435